.codesandbox/ci.json

```diff
@@ -2,5 +2,5 @@
   "buildCommand": "build:codesandbox",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26928&commitId=e91af7f359b742a51ef6d9d76a150d3a3e2d2830&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1038 screenshots unchanged, 0 screenshots with different dimensions and 7 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Normal Overflow selected.default.chromium.png|31| Changed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|43| Changed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|15| Changed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|27| Changed|
|PeoplePicker.Normal selected.default.chromium.png|15| Changed|
|PeoplePicker.Normal selected.suggestions.chromium.png|27| Changed|
|PeoplePicker.Normal.suggestions.chromium.png|12| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>